### DISCUSSION TITLE
[A-L] (2023/24) Foglio 13 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#280] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/2f9cabb9-a9c0-4289-89e2-ea41dcd039a9)

**Comment by Elia-Belli** [comment]
- Ricordiamo la seguente proposizione che ci garantisce l'esistenza dell'applicazione:
 siano $V,W$ spazi vettoriali, $\{v_1,...,v_n\}\subseteq V$ e $\{w_1,...,w_n\}\subseteq W$, allora $\exists!\ T:V\rightarrow W\ lineare\ |\ T(v_i)=w_i$

   Mentre il seguente corollario ci mostra che l'applicazione è univocamente determinata dai valori che assume su una base:
siano $S,T:V\rightarrow W$ lineari, dalla proposizione sappiamo che $S(v_i)=T(v_i)$, se $\{v_1,...,v_n\}$ è base di $V\Rightarrow S=T$ 

  Quindi ossevando che i vettori $(1,1,1),(0,1,1),(0,1,-1)$ sono linearmente indipendenti e quindi base di $\mathbb{R}^3$, allora si applicano le proposizioni precedenti.

- Riscriviamo i vettori della base canonica come combinazioni lineare dei vettori $v_1=(1,1,1),v_2=(0,1,1),v_3=(0,1,-1)$:
  $e_1=(1,0,0)=v_1-v_2\Rightarrow F(e_1)=F(v_1-v_2)=(2,3,2)-(1,3,2)=(1,0,0)$
  $e_2=(0,1,0)=\frac{1}{2}v_2+\frac{1}{2}v_3\Rightarrow F(e_2)=F(\frac{1}{2}v_2+\frac{1}{2}v_3)=\frac{1}{2}(1,3,2)+\frac{1}{2}(1,1,-2)=(1,2,0)$
  $e_3=(0,0,1)=\frac{1}{2}v_2-\frac{1}{2}v_3\Rightarrow F(e_3)=F(\frac{1}{2}v_2-\frac{1}{2}v_3)=\frac{1}{2}(1,3,2)-\frac{1}{2}(1,1,-2)=(0,1,2)$

**Comment by daromacarlo** [comment]
![Immagine 2023-12-05 170558](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/2817d666-efc2-41bf-aff7-76278dfbcaeb)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale

![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/10424eb2-728f-40ce-aa6b-5404ada46043)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/e507fbec-1eee-43b3-92a8-cd40fff6ec2d)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/2143f23e-b84c-441e-9591-5110775bd138)
